### PR TITLE
post-processor/shell-local: don't set executable bit for artifact files

### LINKED ISSUE #3504: shell-local post processor set wrong permission on artifact files
i'm compile packer from current master branch
```
==> qemu: Running post-processor: shell-local
==> qemu (shell-local): Post processing with local shell script: chmod +x /tmp/packer-shell947597411 output/centos-7-x86_64/centos-7-x86_64; PACKER_BUILD_NAME='qemu' PACKER_BUILDER_TYPE='qemu' /tmp/packer-shell947597411 output/centos-7-x86_64/centos-7-x86_64
```

As you see - chmod +x runs on script and on artifact file, this is wrong.

Corresponding json file:
```
{
        "type": "shell-local",
        "inline": ["mv $1 {{user `DESTDIR` }}/"]
      }
```

### PR DESCRIPTION
**Pull request by vtolstov**
Closes #3504

Signed-off-by: Vasiliy Tolstov <v.tolstov@selfip.ru>

### CODE REVIEW

**Comment by vtolstov** [comment]
What about this fix? Because it important for me (i'm restore permissions on destination and get errors)

File: post-processor/shell-local/post-processor.go [hunk 1]
```diff
@@ -65,7 +67,7 @@ func (p *PostProcessor) Configure(raws ...interface{}) error {
 	}
 
 	if p.config.ExecuteCommand == "" {
-		p.config.ExecuteCommand = "chmod +x {{.Path}}; {{.Vars}} {{.Path}}"
+		p.config.ExecuteCommand = "chmod +x {{.Script}}; {{.Vars}} {{.Script}} {{.Artifact}}"
```
Comment (cbednarski) [inline post-processor/shell-local/post-processor.go]: Can you make a note in the docs to reflect this fix? Also please note the issue you ran into so if someone wants to customize this command they do not make the same mistake we did.

**Comment by cbednarski** [comment]
This is subtle! Good catch and thanks for the PR! This needs a small update to the docs and then we're ready to merge.

**Comment by vtolstov** [comment]
I'm fix the docs, i don't think that we need to add additional note about this problem, because now variables have names that hard to mix

**Comment by cbednarski** [comment]
Looks great, thanks!

**Pull request merged**